src/transformers/models/llava_next_video/modeling_llava_next_video.py

```diff
@@ -853,7 +853,7 @@ def forward(
             inputs_embeds = self.get_input_embeddings()(input_ids)
 
         # Merge text and images in prefill stage
-        if past_key_values is None:
+        if input_ids.shape[1] != 1:
```

Comment:
nope, we don't expect a user to pass inputs embeds and pixels at the same time, because for merging we need to slice image token ids positions 

Yes, we have a ValueError in case both are passed, but it is in latest `main`. IMO we don't need error in this PR, users even don't try to pass embeds in llava 😄